public/kibi_timeline.js

```diff
@@ -149,25 +149,29 @@ define(function (require) {
         const groupId = group.id;
         const groupColor = group.color;
 
+        searchSource.highlight({
```

Comment:
I think you should use highlight tags introduced by kibana:

- https://github.com/sirensolutions/kibi/blob/master/src/ui/public/highlight/highlight_tags.js
- https://github.com/sirensolutions/kibi/blob/master/src/ui/public/highlight/highlight.js